src/components/attendee/AttendeeLayout.tsx

```diff
@@ -69,6 +93,8 @@ const _AttendeeLayout: React.FC<_AttendeeLayoutProps> = ({ userId }) => {
 export const AttendeeLayout = compose(
   withAuth,
   withFallback(["isAuthLoaded", "userId"], NotLoggedInFallback),
+  withSlugs,
+  withSpacesBySlug,
   withRequired({
     required: ["userId"],
     fallback: LoadingPage,
```

Comment:
Most likely [OOS] and if true, the following is just FYI and maybe a request to start putting `@debt` comments above these `componse()` cases.

I think we should replace this with a more pedestrian HOC alternative. I already had something in a hooks redesing for this component, but we do have few example already in this branch with the extension `.hoc.tsx` such as in https://github.com/sparkletown/sparkle/blob/redesign-2021/src/pages/auth/Login/Login.hoc.tsx